bundle/edu.gemini.dataman.app/src/main/scala/edu/gemini/dataman/app/Dataman.scala

```diff
@@ -98,7 +113,8 @@ object Dataman {
       schedulePoll(pollArchive.thisWeek, config.thisWeekPeriod)
 
       schedule(progSync, progSyncDelay, config.allPeriod.time)
-      schedule(new RetryFailedRunnable(User, retryMinDelay, exec, odb), retryMinDelay.plusMillis(1), retryPeriod)
+      schedule(new ObsRefreshRunnable(odb, User, pollService), obsRefreshDelay, config.obsRefreshPeriod.time)
```

Comment:
The `ObsRefreshRunnable` will periodically scan the ODB looking for observations that have datasets which we expect will change "soon".  So if a given archive poll that results from that fails, it will be retried in the next scan if the datasets are still expected to be updated.